package.json

```
@@ -39,25 +39,27 @@
     "node": ">=12.0.0",
     "npm": ">=6.0.0"
   },
+  "eslintConfig": {
+    "extends": "ipfs"
+  },
   "homepage": "https://github.com/libp2p/js-libp2p-kad-dht",
   "dependencies": {
```

We need to let tsc know where to look for the types

```suggestion
  "types": "dist/src/index.d.ts",
  "dependencies": {
```